cloudbuild.yaml

```diff
@@ -14,10 +14,118 @@
 
 
 steps:
-# Uses the docker build step to build an image called cloud-run-pubsub-service
-- name: 'gcr.io/cloud-builders/docker'
-  args: ['build', '-t', 'gcr.io/$PROJECT_ID/cloud-run-pubsub-service', './philips_hue_integration_example']
+- id: 'build docker image'
+  # Uses the docker build step to build an image
+  name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/${_IMAGE_NAME}', './philips_hue_integration_example']
 
-# cloud-run-pubsub-service is pushed to Container Registry
+- id: 'branch name'
```

Comment:
Is this step just to output variables? Maybe a name like `print branch name`.

Also, I think you can just use the `bash` step? See: https://cloud.google.com/cloud-build/docs/configuring-builds/run-bash-scripts#running_inline_bash_scripts

Reply:
Yeah, this step is just to output the current GitHub branch. I changed the id to be more clear.